src/main/java/org/jenkinsci/plugins/deployment/HostRecords.java

```diff
@@ -0,0 +1,21 @@
+package org.jenkinsci.plugins.deployment;
+
+import hudson.model.InvisibleAction;
+
+import java.util.Collection;
+
+public class HostRecords extends InvisibleAction {
+    Collection<HostRecord> hostrecords;
+
+    public HostRecords(Collection<HostRecord> hostRecords) {
+        this.hostrecords = hostRecords;
+    }
+
+    public Collection<HostRecord> getHostrecords() {
+        return hostrecords;
+    }
+
+    public void setHostrecords(Collection<HostRecord> hostrecords) {
```

Comment:
Method is unused. It's better to delete it. Otherwise there will be a potential issue when somebody saves the action without patching the build